pkg/apis/apps/kfdef/v1/application_types.go

```diff
@@ -54,8 +54,25 @@ type KfDefSpec struct {
 	Plugins      []Plugin      `json:"plugins,omitempty"`
 	Secrets      []Secret      `json:"secrets,omitempty"`
 	Repos        []Repo        `json:"repos,omitempty"`
+	Global       Global        `json:"global,omitempty"`
 }
 
+type Global struct {
+	Transformers []Transformer `json:"transformers,omitempty"`
+	Generators   []Generator   `json:"generators,omitempty"`
+}
+
+type Transformer struct {
+	Name    string   `json:"name,omitempty"`
+	RepoRef *RepoRef `json:"repoRef,omitempty"`
+}
+
+type Generator struct {
```

Comment:
Since you are only adding the transformers implementation, should we leave out the Generator section here? Or maybe have it there but commented out for now?

Reply:
Yes, I will comment out the generators section